.github/condarc

```diff
@@ -4,3 +4,4 @@ notify_outdated_conda: False
 changeps1: False
 always_yes: True
 local_repodata_ttl: 7200
+channel_priority: strict
```

Comment:
```suggestion
channel_priority: strict
```

This should only apply to the CI step of provisioning `base`. Otherwise it'll affect all tests.